mineral-ui/mineral-ui
# [Site] Add Theming page

### Description

- Flesh out content in Theming page, including table of theme variables
- Add `colorable`
- Add CodeBlock component for Site

### Motivation and context

connects #56 

### How to test


1. `cd mineral-ui/packages/site && npm start`
2. Navigate to Theming page
3. Check that the content is good

### Types of changes

- Other (provide details below)

Docs

### Checklist




### How does this PR make you feel?

![Page](https://media.giphy.com/media/zuNdVweFYGXy8/giphy.gif)
